docs/manual/execution-lifecycle/job-data.md

```diff
@@ -0,0 +1,202 @@
+# Job Data Plugins (Enterprise) [Incubating]
+::: enterprise
+:::
+
+::: incubating
+:::
+
+This plugin allows Jobs to export a JSON file as the result of an execution.  The JSON data can be retrieved via the API, and shown in the GUI.
```

Comment:
Is it really returning a file or does it export JSON "data"?

Reply:
what's the difference?

Reply:
file, to me, implies "download this file" versus data that might be processed programmatically.